### ISSUE TITLE
Not reproducible test recordings

### ISSUE THREAD

**Issue by simlu** [tristanls/dynamodb-lock-client#6]
We are mocking all our tests outgoing requests (using lambda-tdd). However since the owner is different per environment we can't really use mocking (i.e. ci will have a different owner than local especially since we use docker for everything)

Looking in particular at the line
```js
owner: `${pkg.name}@${pkg.version}_${os.userInfo().username}@${os.hostname()}
```
https://github.com/tristanls/dynamodb-lock-client/blob/master/index.js#L50

Any reason why we need the `os...` information here? My preference would be to make this customization at the very least or even just change this to `${pkg.name}@${pkg.version}`. Thoughts? I will create a pr if you are ok with changing this.

Edit: Create a pr here https://github.com/tristanls/dynamodb-lock-client/pull/7/files

**Comment by simlu** [comment]
I've done a pr now that allows you to customise the owner here. Would much appreciate if we could push this forward!

https://github.com/tristanls/dynamodb-lock-client/pull/8/files

Cheers!

**Comment by tristanls** [comment]
Thank you @simlu , I didn't get a chance to come back to this till now. Released your changes in [v0.3.0](https://github.com/tristanls/dynamodb-lock-client/releases/tag/v0.3.0).

**Comment by simlu** [comment]
Sweet, thank you very much @tristanls !
